src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/jpa/PrisonerIepLevel.kt

```diff
@@ -35,5 +35,5 @@ data class PrisonerIepLevel(
 }
 
 enum class ReviewType {
-  INITIAL, REVIEW, TRANSFER, ADJUSTMENT, MIGRATED,
```

Comment:
what was `ADJUSTMENT` for? i can't remember

Reply:
We aren't using it so felt like it was bad to leave it there!